src/saml2/response.py

```diff
@@ -409,11 +409,17 @@ def _verify(self):
                 raise RequestVersionTooHigh()
 
         if self.asynchop:
-            if (
-                self.response.destination
-                and self.response.destination not in self.return_addrs
+            if not (
+                 getattr(self.response, 'destination')
             ):
-                logger.error("%s not in %s", self.response.destination, self.return_addrs)
+                logger.error(
+                    f"Invalid response destination in asynchop"
```

Comment:
no f string necessary

Reply:
done here https://github.com/peppelinux/pysaml2/commit/1476eb93c622b82d866464c156ab66ae0c059b83
thnak you, I completely agree